scala/ppml/src/test/scala/com/intel/analytics/bigdl/ppml/crypto/dataframe/ParquetEncryptionSpec.scala

```diff
@@ -32,10 +32,9 @@ class ParquetEncryptionSpec extends DataFrameHelper {
   val (plainFileName, encryptFileName, data, dataKeyPlaintext) = generateCsvData()
 
   val ppmlArgs = Map(
-    "spark.bigdl.kms.appId" -> appid,
-    "spark.bigdl.kms.appKey" -> apikey,
-    "spark.bigdl.kms.primaryKey" -> primaryKeyPath,
-    "spark.bigdl.kms.dataKey" -> dataKeyPath
+      "spark.bigdl.primaryKey.defaultKey.kms.appId" -> appid,
+      "spark.bigdl.primaryKey.defaultKey.kms.apiKey" -> apikey,
+      "spark.bigdl.primaryKey.defaultKey.material" -> (s"file://" + primaryKeyPath),
```

Comment:
Do not use file:// before key path. That's will force key path become local path. Let's primary key path include file prefix, e.g., hdfs:// or s3://.

Reply:
going to check